### PR TITLE
[DPEDE-625] Improve ChiDataTableFilters odd behavior when there are several data tables

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**
https://ctl.atlassian.net/browse/DPEDE-620

### CODE REVIEW

File: src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx [hunk 1]
```diff
@@ -386,8 +375,8 @@ export default class AdvancedFilters extends Vue {
     this.$emit(DATA_TABLE_EVENTS.ADVANCED_FILTERS_CHANGE);
   }
 
-  async _applyAdvancedFiltersChange() {
-    await this.storeModule.updateFilterConfig(this.filterElementValueLive);
+  _applyAdvancedFiltersChange() {
+    // this.storeModule.updateFilterConfig(this.filterElementValueLive);
```
Comment (SergioQCL) [inline src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx:379]: Is this needed?

File: src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx [hunk 2]
```diff
@@ -403,9 +392,19 @@ export default class AdvancedFilters extends Vue {
     }
   }
 
-  async _resetAdvancedFilters() {
-    await this.storeModule.updateFilterConfig({ ...this.filterElementValue, ...this._planeAdvancedData });
-    await this.storeModule.updateFilterConfigLive({ ...this.filterElementValueLive, ...this._planeAdvancedData });
+  _resetAdvancedFilters() {
+    if (this.advancedFiltersData) {
```
Comment (SergioQCL) [inline src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx:396]:
```suggestion
    if (!this.advancedFiltersData) {
    return;
    }
```

File: src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx [hunk 3]
```diff
@@ -403,9 +392,19 @@ export default class AdvancedFilters extends Vue {
...
+      this.advancedFiltersData?.forEach((currentValue: any) => {
```
Comment (SergioQCL) [inline src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx:397]:
```suggestion
      this.advancedFiltersData?.forEach((currentValue: DataTableFilter) => {
```

File: src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx [hunk 4]
```diff
@@ -70,12 +72,15 @@ export default class DataTableFilters extends Vue {
 
     this._advancedFiltersData = copyArrayOfObjects(advancedFilters);
 
-    const plainFiltersData = this.$props.filtersData.reduce((accumulator: any, currentValue: any) => {
-      return { ...accumulator, [currentValue.id]: currentValue.type === 'checkbox' ? false : currentValue.value || '' };
-    }, {});
+    this.$props.filtersData.forEach((currentValue: any) => {
```
Comment (SergioQCL) [inline src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx:75]: Is there a better type here?
```suggestion
    this.$props.filtersData.forEach((currentValue: any) => {
```

File: src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx [hunk 5]
```diff
@@ -307,10 +298,10 @@ export default class DataTableFilters extends Vue {
...
   }
 
-  async applyChanges() {
-    await this.storeModule.updateFilterConfig({
-      ...this.filterElementValueLive,
-    });
+  applyChanges() {
+    // this.storeModule.updateFilterConfig({
+    //   ...this.filterElementValueLive,
+    // });
```
Comment (SergioQCL) [inline src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx:304]: Needed?

**Review by SergioQCL — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**